.github/workflows/unit-tests-recipes.yml

```
@@ -129,6 +129,7 @@ jobs:
     needs: changed-dirs
     runs-on: linux-amd64-gpu-l4-latest-1
     if: ${{ needs.changed-dirs.outputs.dirs != '[]' }}
+    name: "unit-tests (${{ replace(matrix.recipe.dir, 'bionemo-recipes/', '') }})"
```

actionlint (1.7.7)

undefined function "replace". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)